staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -1053,6 +1053,19 @@ message WindowsContainerResources {
     int64 memory_limit_in_bytes = 4;
     // Specifies the size of the rootfs / scratch space in bytes to be configured for this container. Default: 0 (not specified).
     int64 rootfs_size_in_bytes = 5;
+    // Specifies the set of CPUs to affinitize for this container.
```

Comment:
Can we add some notes on what cri implementation must do when this structure contains incorrect data.

Reply:
CRI would basically just forward these values to the shim via oci runtime spec. hcsshim for example will error out while trying to set the affinity values if they are invalid. Are you suggesting we summarize this in the comment here?

cc @jsturtevant